modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json

```
@@ -937,6 +937,30 @@
         "source_name": "EVA",
         "source_version": 4
     },
+    {
+      "id": "eva_nomascus_leucogenys_gca000146795v3",
+      "description": "Variants from EVA",
+      "species": "nomascus_leucogenys",
```

The species name needs to be capitalised or not really?

```suggestion
      "species": "Nomascus_leucogenys",
```

---

The one place I know it use do `lc` - https://github.com/Ensembl/ensembl-variation/blob/d34d25e481f22c11705b23a1aded34feb8434010/modules/Bio/EnsEMBL/Variation/DBSQL/VCFCollectionAdaptor.pm#L144

Do you know why it would need capitalizing?

---

I am asking because in other species it's capitalised, but as long as it works, I'm fine with it.